model/flow/chunk.go

```diff
@@ -4,9 +4,10 @@ type ChunkBody struct {
 	CollectionIndex uint
 
 	// execution info
-	StartState      StateCommitment // start state when starting executing this chunk
-	EventCollection Identifier      // Events generated by executing results
-	BlockID         Identifier      // Block id of the execution result this chunk belongs to
+	StartState           StateCommitment // start state when starting executing this chunk
+	StateDeltaCommitment Identifier      // a commitment over a set of state deltas
+	EventCollection      Identifier      // Events generated by executing results
+	BlockID              Identifier      // Block id of the execution result this chunk belongs to
```

Comment:
Last time there was a change here, it broke some third-party apps computing the hash of chunks and the hash of results. We need to check this is not going to happen again.